scripts/lib/api/updateLinks.test.ts

```diff
@@ -212,7 +212,15 @@ describe("relativizeLink()", () => {
       "https://qiskit.org/documentation/apidoc/qiskit.circuit.BreakLoopOp.html#qiskit.circuit.BreakLoopOp",
       "/api/qiskit/qiskit.circuit.BreakLoopOp",
     ],
-  ])("transform qiskit.org Qiskit API links", (input, expected) => {
+  ])("relativize qiskit.org Qiskit API links", (input, expected) => {
+    expect(relativizeLink({ url: input })).toEqual({ url: expected });
+  });
+
+  test.each([
+    ["https://docs.quantum.ibm.com/run", "/run"],
+    ["https://docs.quantum.ibm.com/api/qiskit/utils", "/api/qiskit/utils"],
+    ["https://docs.quantum-computing.ibm.com/run", "/run"],
```

Comment:
```suggestion
    ["https://docs.quantum-computing.ibm.com/run", "/run"],
    ["https://docs.quantum-computing.ibm.com", "/"],
```

Probably an edge case, but should we handle this?

Reply:
Good thinking. I think it's fine though. The script would do nothing for `https://docs.quantum-computing.ibm.com` (good), whereas it would transform `https://docs.quantum-computing.ibm.com/` to `/`. But that's not a valid synthetic file, so the link checker would error:

https://github.com/Qiskit/documentation/blob/566b553a243ac9801890157950350ea498735137/scripts/commands/checkLinks.ts#L25-L28

I can add a test that `https://docs.quantum-computing.ibm.com` without the `/` is ignored.

Reply:
I guess I meant "Should we relativize `https://docs.quantum-computing.ibm.com`"? But it's probably rare enough that I think it's fine either way.